lib/badwords.json

```diff
@@ -1,61 +1,87 @@
 [
+    "abbo",
+    "abo",
     "beeyotch",
     "biatch",
     "bitch",
     "chinaman",
     "chinamen",
     "chink",
+    "coolie",
+    "coon",
     "crazie",
     "crazy",
     "crip",
+    "cuck",
     "cunt",
     "dago",
     "daygo",
     "dego",
     "dick",
-    "dumb",
     "douchebag",
+    "dumb",
     "dyke",
     "fag",
+    "faggot",
     "fatass",
     "fatso",
     "gash",
     "gimp",
     "golliwog",
     "gook",
+    "goy",
+    "goyim",
     "gyp",
-    "halfbreed",
+    "gypsy",
     "half-breed",
+    "halfbreed",
+    "heeb",
     "homo",
     "hooker",
     "idiot",
     "insane",
     "insanitie",
     "insanity",
     "jap",
+    "kaffer",
+    "kaffir",
```

Comment:
@dariusk  I mentioned this project on MetaFilter in [a discussion about automatically filtering for a certain slur](https://metatalk.metafilter.com/25598/Feature-Request-prevent-racial-slurs-from-being-posted). Per [this comment](https://metatalk.metafilter.com/25598/Feature-Request-prevent-racial-slurs-from-being-posted#1371843) I second the request for at least this line to be added to the list.